### DISCUSSION TITLE
Installing Kibanana dashboards on securityonion 2.3.130 deployment

### DISCUSSION THREAD

**Discussion by HuntingSecurity** [Security-Onion-Solutions/securityonion#8514] [Unsupported Versions]
Good evening,

I am currently running securityonion 2.3.130 and won't be able to update to .140 for at least a few more weeks. I've been looking into elastic dashboards and found the beaconing detection dashboards to be quite interesting and would like to incorporate that into my deployment.  I am specifically looking at the beaconing.md release from elastic. Currently running 1 master and between 1-6 sensors depending on the situation. I have looked over the elastic documentation but it doesn't seem to fit into uploading the dashboards into a security onion deployment. Looking for anyhelp getting these dashboards thrown into my deployment until I can update to .140.

I am still semi-new to working with security onion so if I'm just completely misunderstanding the elastic documentation I apologize.

**Comment by weslambert** [comment]
You should be able to import dashboards by navigating to `Stack Management` -> `Saved Objects` -> `Import`  (from within Kibana).
